Automattic/wp-calypso
# Untangling: Add Settings -> Web server subtab

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9614

## Proposed Changes

This PR adds the Settings -> Web server subtab under the `untangling/hosting-menu` flag.

<img width="841" alt="image" src="https://github.com/user-attachments/assets/8f8a94a7-d7b2-429c-a216-25036889cce2">

The settings come from Server Settings -> {Web server settings, Defensive mode} cards.

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Settings -> Web server subtab.
    - Verify that this setting is working.
5. Click a Simple site from the site list
    - Verify that this setting is not available.
7. Regression testing:
    - Go to Server Settings
    - Verify that the original Web Server Settings and Defensive Mode cards are still working.

> [!NOTE]  
> We will address any UI polish in the next PRs.

## Pre-merge Checklist
